jax/core.py

```diff
@@ -202,6 +202,19 @@ def gensym(jaxprs: Optional[Sequence[Jaxpr]] = None,
   counter = it.count(start=start)
   return lambda aval: Var(next(counter), suffix, aval)
 
+# In a jaxpr, `dropvar` can appear in place of a bound variable to indicate that
+# the assignment is dropped, i.e. that an expression's output value will never
+# be read. In that sense, `dropvar` is not a variable, but it is convenient to
+# treat it as a special case of one.
+class _DropVar(Var):
```

Comment:
```suggestion
class DropVar(Var):
```

The underscore can show up in pdb sessions and the like, and looks a bit ugly. I think we already protect this namespace, and e.g. have internal classes that aren't prefixed with underscores, so IMO it is optional